gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/commands/AbstractHystrixCommand.groovy

```diff
@@ -49,7 +53,11 @@ abstract class AbstractHystrixCommand<T> extends HystrixCommand<T> {
 
   @Override
   protected T run() throws Exception {
-    return work()
+    try {
+      return work()
+    } catch (RetrofitError error) {
+      throw UpstreamBadRequest.classifyError(error, [HttpStatus.TOO_MANY_REQUESTS.value()])
```

Comment:
why not just propagate any 4xx back to the caller since we generally don't have any smarts in this layer?

if I submit malformed JSON or ask for a missing resource that should come back out